### PR TITLE
chore: Migrate to zeit now deployments

### PR DESCRIPTION
**Pull request by doniyor2109**

### CODE REVIEW

**Comment by doniyor2109** [comment]
@umidbekkarimov Where can I configure domain DNS configuration?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -53,14 +53,55 @@ jobs:
...
-          yarn docs --quiet
-          yarn firebase deploy --token ${{secrets.FIREBASE_TOKEN}}
-
       - name: Publish
         if: ${{ startsWith(github.ref, 'refs/tags/') }}
         run: |
           echo "//registry.npmjs.org/:_authToken=${{secrets.NPM_TOKEN}}" > ./.npmrc
           yarn lerna publish from-git --yes
+
+  build-docs:
```
Comment (umidbekkarimov) [inline .github/workflows/build.yml:62]: why do we need separate step?

Comment (doniyor2109) [reply .github/workflows/build.yml:62]: So we do not wait for builds

**Comment by umidbekkarimov** [comment]
@doniyor2109 why do we have a new comment after each commit?

**Comment by umidbekkarimov** [comment]
> @umidbekkarimov Where can I configure domain DNS configuration?

https://console.cloud.google.com/

**Comment by doniyor2109** [comment]
> @doniyor2109 why do we have a new comment after each build?

@umidbekkarimov I guess zeit action does not update comments. I am researching..

**Comment by doniyor2109** [comment]
@umidbekkarimov Some how zeit cannot find previous comment 
https://github.com/amondnet/now-deployment/blob/965864136f70b890b6c864b223f638fd5914dff9/index.js#L141

**Comment by umidbekkarimov** [comment]
@doniyor2109 also each comment creates a new hash, so seems useless

File: .github/workflows/build.yml (hunk 1 above)
Comment (umidbekkarimov) [reply .github/workflows/build.yml:62]: but it's harder to maintain workflow file now

Comment (doniyor2109) [reply .github/workflows/build.yml:62]: Can we reuse those setup steps? If not I will revert them

**Comment by umidbekkarimov** [comment]
@doniyor2109, unfortunately, we can't use it, it creates too much spam, we need official integration

**Pull request closed without merging**